### PR TITLE
Update Ukrainian translation

### PR DESCRIPTION
**Pull request by otkd**
Builds on excellent effort by @djperya and @vickas777 with some minor refinements and updated .mo which wasn't in https://github.com/SoftFever/OrcaSlicer/pull/1717

### CODE REVIEW

File: bbl/i18n/uk/OrcaSlicer_uk.po [hunk 1]
```diff
@@ -7352,8 +7354,8 @@ msgid ""
 "has slight overflow or underflow"
 msgstr ""
 "Матеріал може мати об'ємну зміну після перемикання між розплавленим станом "
-"та кристалічним станом. Ця настройкапропорційно змінює весь екструзійний "
-"потік цієї нитки розжарювання в gcode.Рекомендований діапазон значень - від "
+"та кристалічним станом. Ця настройка пропорційно змінює весь екструзійний "
```
Comment (vickas777) [inline bbl/i18n/uk/OrcaSlicer_uk.po:7357]:
```suggestion
"та кристалічним станом. Це налаштування пропорційно змінює весь екструзійний "
```

File: bbl/i18n/uk/OrcaSlicer_uk.po [hunk 2]
```diff
@@ -7060,8 +7062,8 @@ msgid ""
 "A gap between innermost brim line and object can make brim be removed more "
 "easily"
 msgstr ""
-"Зазор між внутрішньою лінією кайми та предметом може сприяти "
-"більшелегкого зняття кайми"
+"Зазор між внутрішньою лінією кайми та предметом може сприяти більшелегкого "
```
Comment (vickas777) [inline bbl/i18n/uk/OrcaSlicer_uk.po:7065]:
```suggestion
"Зазор між внутрішньою лінією кайми та предметом може сприяти легшому "
```

File: bbl/i18n/uk/OrcaSlicer_uk.po [hunk 3]
```diff
@@ -7060,8 +7062,8 @@ msgid ""
...
+"зняття кайми"
```
Comment (vickas777) [inline bbl/i18n/uk/OrcaSlicer_uk.po:7066]:
```suggestion
"зняттю кайми"
```

File: bbl/i18n/uk/OrcaSlicer_uk.po [hunk 4]
```diff
@@ -5320,30 +5321,31 @@ msgid ""
 "settings: at least 2 interface layers, at least 0.1mm top z distance or "
 "using support materials on interface."
 msgstr ""
-"Для стилів \"Tree Strong\" та \"Tree Hybrid\" ми рекомендуємо "
-"наступнеПараметри: не менше 2 інтерфейсних шарів, не менше 0,1 мм відстань "
-"між вершинами z або використання допоміжних матеріалів за інтерфейсом."
+"Для стилів \"Tree Strong\" та \"Tree Hybrid\" ми рекомендуємо наступне "
```
Comment (vickas777) [inline bbl/i18n/uk/OrcaSlicer_uk.po:5324]:
```suggestion
"Для стилів \"Tree Strong\" та \"Tree Hybrid\" ми рекомендуємо наступні "
```

File: bbl/i18n/uk/OrcaSlicer_uk.po [hunk 5]
```diff
@@ -5320,30 +5321,31 @@ msgid ""
...
+"параметри: не менше 2 інтерфейсних шарів, не менше 0,1 мм відстань між "
+"вершинами z або використання допоміжних матеріалів за інтерфейсом."
```
Comment (vickas777) [inline bbl/i18n/uk/OrcaSlicer_uk.po:5326]:
```suggestion
"вершинами z або використання допоміжних матеріалів в якості інтерфейсу."
```

File: bbl/i18n/uk/OrcaSlicer_uk.po [hunk 6]
```diff
@@ -2357,17 +2357,18 @@ msgid ""
...
+"\n"
 
 #, c-format, boost-format
 msgid ""
 "Recommended nozzle temperature of this filament type is [%d, %d] degree "
 "centigrade"
 msgstr ""
-"Рекомендована температура сопла для цього типу нитки становить [%d, "
-"%d]градусів Цельсія"
+"Рекомендована температура сопла для цього типу нитки становить [%d, %d] "
```
Comment (vickas777) [inline bbl/i18n/uk/OrcaSlicer_uk.po:2370]: Use one translation for nozzle everywhere. L2360 uses different translation than current

Comment (djperya) [reply bbl/i18n/uk/OrcaSlicer_uk.po:2370]: Thanks for joining!

Comment (otkd) [reply bbl/i18n/uk/OrcaSlicer_uk.po:2368]: Missed that one, thank you! Fixed in a6dcadc5198ff3d6449d3a6c32224e25c0c45a45

**Comment by otkd** [comment]
@vickas777 thank you for suggestions, have added them and updated

**Comment by SoftFever** [comment]
Hi all,

Thank you so much!
Could you solve the conflict so that I can merge it?

**Comment by otkd** [comment]
> Hi all,
> 
> Thank you so much! Could you solve the conflict so that I can merge it?

@SoftFever done, thank you!

**Review by SoftFever — APPROVED** [review]

**Comment by SoftFever** [comment]
Thank you.
Merged

**Pull request merged**